iac-common/src/main/java/org/sonar/iac/common/api/checks/CheckContext.java

```diff
@@ -19,10 +19,11 @@
  */
 package org.sonar.iac.common.api.checks;
 
-import java.util.List;
 import org.sonar.iac.common.api.tree.HasTextRange;
 import org.sonar.iac.common.api.tree.impl.TextRange;
 
+import java.util.List;
```

Comment:
I wonder why imports are not rearranged like this on my machine. Do you have `optimize imports` with an `importorder` file enabled in the Eclipse formatter adapter?

Reply:
I will check with other squad member so that I can align 👍